### PR TITLE
Add boilerplate information to README

### PR DESCRIPTION
**Pull request by juliusgeo**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,11 +1,72 @@
...
+This package provides an ``ExplainCollection`` class that allows PyMongo's Collection methods to be explained_
 
 .. _explained: https://docs.mongodb.com/master/reference/command/explain/#dbcmd.explain.
 
 
+Support / Feedback
+==================
+
+For issues with, questions about, or feedback for PyMongoExplain, please look into
+our `support channels <http://www.mongodb.org/about/support>`_. Please
```
Comment (prashantmital) [inline README.rst:21]: I am not sure that we can mention our official support channels here. I don't believe this package will have the same level of support as the official driver. @ShaneHarvey @behackett thoughts?

File: README.rst [hunk 2]
```diff
@@ -1,11 +1,72 @@
...
+questions - you're more likely to get an answer on the `mongodb-user
+<http://groups.google.com/group/mongodb-user>`_ list on Google Groups.
+
+Bugs / Feature Requests
+=======================
+
+Think you’ve found a bug? Want to see a new feature in PyMongoExplain?
+Please open a case in our issue management tool, JIRA:
+
+- `Create an account and login <https://jira.mongodb.org>`_.
```
Comment (prashantmital) [inline README.rst:32]: We might want to consider letting people open issues against pymongoexplain in GitHub itself? We do need to figure out where we want to track pymongoexplain-related issues in the long-term. CC: @ShaneHarvey

File: README.rst [hunk 3]
```diff
@@ -1,11 +1,72 @@
...
+-------------------
+
+Please include all of the following information when opening an issue:
+
+- Detailed steps to reproduce the problem, including full traceback, if possible.
+- The exact python version used, with patch level::
+
+  $ python -c "import sys; print(sys.version)"
+
+- The exact version of PyMongo used (if applicable), with patch level::
```
Comment (prashantmital) [inline README.rst:49]: In this case we would ask for the version of `pymongoexplain` would we not?

File: README.rst [hunk 4]
```diff
@@ -1,11 +1,72 @@
...
+  $ python -c "import pymongo; print(pymongo.version); print(pymongo.has_c())"
+
+
+
+Dependencies
+============
+
+PyMongoExplain supports CPython 3.5+, and PyPy3.5+.
+
+PyMongoExplain requires `PyMongo <https://github.com/mongodb/mongo-python-driver/>`_
```
Comment (prashantmital) [inline README.rst:60]: This should mention the minimum version of PyMongo required for `pymongoexplain`.

File: README.rst (hunk 1 above)
Comment (ShaneHarvey) [reply README.rst:21]: Agree. Let's just mention the community site:
```
For questions, discussions, or general technical support, visit the MongoDB Community Forums. The MongoDB Community Forums are a centralized place to connect with other MongoDB users, ask questions, and get answers.
```

File: README.rst (hunk 2 above)
Comment (ShaneHarvey) [reply README.rst:32]: Let's just use github issues here. We're already using them to track issues and improvements.

File: README.rst (hunk 1 above)
Comment (prashantmital) [reply README.rst:21]: This sounds good to me.

File: README.rst (hunk 2 above)
Comment (prashantmital) [reply README.rst:32]: Agreed. Lets stick with GitHub for now. Julius - please update this text to ask users to file issue tickets under the GitHub project itself. Please remove all references to JIRA.

File: README.rst (hunk 3 above)
Comment (prashantmital) [reply README.rst:49]: Please modify this to ask for versions of both `pymongo` and `pymongoexplain`

File: README.rst (hunk 4 above)
Comment (prashantmital) [reply README.rst:60]: Please mention that `pymongoexplain` requires PyMongo>=3.11,<4

File: README.rst (hunk 2 above)
Comment (juliusgeo) [reply README.rst:32]: Done.

File: README.rst (hunk 1 above)
Comment (juliusgeo) [reply README.rst:21]: Done.

File: README.rst (hunk 3 above)
Comment (juliusgeo) [reply README.rst:49]: Done.

File: README.rst (hunk 4 above)
Comment (juliusgeo) [reply README.rst:60]: Done.

File: README.rst [hunk 5]
```diff
@@ -52,7 +112,7 @@ Since ``ExplainCollection`` instances provide all the same methods provided by `
 
 
 Explaining commands in a script
```
Comment (ShaneHarvey) [inline README.rst:114]: The explain output for ``update_one`` above looks oddly formatted. Can you fix it?

File: README.rst (hunk 4 above)
Comment (prashantmital) [reply README.rst:59]: Where is this mentioned? I don't see it.

Comment (prashantmital) [reply README.rst:59]: Also, I guess after today's conversation, this actually support PyMongo>=3.10,<4

File: README.rst (hunk 5 above)
Comment (juliusgeo) [reply README.rst:121]: Done.

File: README.rst (hunk 4 above)
Comment (juliusgeo) [reply README.rst]: Done.

File: README.rst [hunk 6]
```diff
@@ -22,37 +82,44 @@ Now you are ready to explain some commands. Remember that explaining a command d
...
+                            {'category': {'$eq': 'apparel'}},
+                            {'quantity': {'$eq': 1057}}
+                            ]
+                        },
+               'stage': 'COLLSCAN', 'direction': 'forward'},
+               'stage': 'UPDATE'}, 'planCacheKey': 'CD8F6D8F',
+               'rejectedPlans': []},
+           'ok': 1.0,
+           'operationTime': Timestamp(1595603051, 3)
+            }
```
Comment (ShaneHarvey) [inline README.rst:115]: The formatting here is still a bit off. Try removing the $clusterTime field.

Comment (juliusgeo) [reply README.rst:115]: Done

Comment (ShaneHarvey) [reply README.rst:115]: Still waiting to see the change.

**Pull request merged**